docs/guides/themes.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How themes work
+category: Guides
+order: 5
+---
+
+## Themes
+
+InstUI provides two main themes.
+
+- [canvas](/#canvas)
+- [canvas-high-contrast](/#canvas-high-contrast)
+
+They are meant to be used together. `canvas` provides 4.5:1 contrast, while `canvas-high-contrast` 7.0:1.
+
+InstUI uses `canvas` as the default theme. You can change the default theme by calling the `.use()` method on the theme object.
+
+```jsx
+---
+type: code
+---
+import { canvasHighContrast } from '@instructure/ui-themes'
+
+canvasHighContrast.use()
+```
+
+or with the `InstUISettingsProvider` component
+
+```jsx
+---
+type: code
+---
+import { canvasHighContrast } from '@instructure/ui-themes'
+// app/component root sets the app theme
+<InstUISettingsProvider theme={canvasHighContrast}>
+  <ASubtreeToBeThemedWithCanvasHighContrast />
+</InstUISettingsProvider>
+```
+
+## Colors
+
+InstUI uses a multi layered color system. These layers are
+
+- Primitives
+- Contrasts
+- UI (colors)
+
+### Primitives
+
+InstUI has a cross-theme (meaning they are the same for every theme) color palette, the `primitives`:
+
+```jsx
+---
+type: example
+---
+<ThemeColors colors={canvas.colors.primitives}></ThemeColors>
+```
+
+Everythig is built using these colors. Ideally, our users should never interact directly with this palette, displayed only for reference.
```

Comment:
Every theme in InstUI is built using these colors. Ideally users should never interact directly with this palette, here its only displayed only for reference.